.readthedocs.yml

```diff
@@ -11,25 +11,26 @@ build:
   os: ubuntu-24.04
   tools:
     python: "3"
+  apt_packages:
+    - graphviz
```

Comment:
I can't figure out how to tell readthedocs to install graphviz.  This doesn't work despite [the documentation](https://docs.readthedocs.io/en/stable/config-file/v2.html#build-apt-packages) suggesting it should.  Any help would be appreciated.